### DISCUSSION TITLE
Data Compliance API

### DISCUSSION THREAD

**Discussion by infinite-looper** [Jericho/ZoomNet#118] [Ideas]
Hi,

It seems like ZoomNet is missing a key API call for data compliance (https://marketplace.zoom.us/docs/api-reference/data-compliance/data-compliance/compliance)

According to Zoom documentation, it is a required call when a user uninstall your application. In theory, it could prevent your App from going live.

**Comment by Jericho** [comment]
Thanks for pointing this out to me, I was not aware of this API endpoint. I thought I had read every single page of their api reference documentation but I never noticed the "Data Compliance" section all the way at the bottom of the left navigation area. I wonder if it's something new or if I just missed it????

Anyway, I will be glad to add a `DataCompliance ` resource to the ZoomNet client.

**Reply by Jericho** [reply]
(see #119)
